index.html

```diff
@@ -124,5 +124,30 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-and-early-access-section">
+    <div id="join-hiring" class="card-subsection">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <a href="https://www.notion.so/Join-the-Pack-b1648f1305a74759b6a0e471cca4a56b"  target="_blank" class="button roles-button">See the roles</a>
+    </div>
+
+    <div id="picture-join">
+      <img id="flying-shoe" class="above-background shake" src="images/joinbox-flying-shoe.png" alt="flying shoe">
+      <img id="pencil-small" class="above-background shake" src="images/joinbox-pencil-small.png" alt="pencil small">
+      <img id="green-box" class="above-background shake" src="images/joinbox-abstract-green-box.png" alt="abstract green box">
+      <img id="blue-ball" class="above-background shake" src="images/joinbox-abstract-blue-ball.png" alt="abstract blue ball">
+      <img id="flying-febby" class="above-background" src="images/joinbox-flying-febby.png" alt="flying febby">
+    </div>
+
+    <div id="early-access" class="card-subsection">
+      <h3> Join for early access </h3>
+      <p> We're gradually opening Ready—get early access to updates and be a part of building the next generation of calendars.</p>
+      <form id="form-early-access">
+        <input type="email" name="email" id="email-early-access" placeholder="Your email" action="/early-access.php" method="post">
```

Comment:
```html
action="" and method="" attributes belong to the <form> element alone.

For <input> elements, you're suppossed to use the formaction="" and formmethod="" attributes.
```

However, I don't understand why you'd put these directly on the input if you don't need to override the values in the form's *action* and *method* attributes (this is what the *formaction* and *formmethod* attributes are usually used for).

You could simply put the *action* and *method* attributes (and their values) in the form, and it would do the same since there's only one input. Code autocompletion does this for you too. I don't see any benefits from going beyond the default approach in this case.